src/process-dir.js

```diff
@@ -1,15 +1,18 @@
 import fs from "fs";
+import * as nodePath from 'path';
```

Comment:
Named `nodePath` to avoid colliding with the existing usage of `path` as a variable name.